docs/reference/sql/endpoints/odbc.asciidoc

```diff
@@ -9,9 +9,12 @@
 [float]
 === Overview
 
-{odbc} is a feature-rich 3.80 ODBC driver for {es}.
-It is a core level driver, exposing all of the functionality accessible through the {es}'s SQL ODBC API, converting ODBC calls into
-{es-sql}.
+{odbc} is a 3.80 compliant ODBC driver for {es}.
+It is a core level driver, exposing all of the functionality accessible through
+the {es}'s SQL ODBC API, converting ODBC calls into {es-sql}.
++
+In order to be able to use of the driver, the server must have the {es-sql}
```

Comment:
`must have {es-sql} installed and running with the valid license`.

Reply:
Thanks, fixed.